dev-env/keys.py

```diff
@@ -0,0 +1,149 @@
+"""Generate keys."""
```

Comment:
how about a separate submodule-repo for this kind of stuff?

Since you have added the redis-cmd.sh to the config-repo and freva-web aswell as freva-nextgen share most of the dev-environment it would make sense to put most of the dev-env config in a separate repo which freva-nextgen and freva-web can use, instead of mirroring everything into to separate repos.

Reply:
Yes, that is a valid point. Since it involves changing two repo's I'd say we do this one first and then add things to the othe one.